### PR TITLE
[SHOT-3291] Align README.md files for tk-vred

### PR DESCRIPTION
**Pull request by 000paradox000**
README file updated according to shotgun standard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![Linting](https://img.shields.io/badge/PEP8%20by-Hound%20CI-a873d1.svg)](https://houndci.com)
 
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+## Documentation
 
-[![Build Status](https://dev.azure.com/shotgun-ecosystem/Toolkit/_apis/build/status/Engines/tk-vred?branchName=master)](https://dev.azure.com/shotgun-ecosystem/Toolkit/_build/latest?definitionId=56&branchName=master)
+This repository is a part of the Shotgun Pipeline Toolkit.
+
+- For more information about this app and for release notes, *see the wiki section*.
```
Comment (rob-aitchison) [inline README.md]: engine, not app

File: README.md [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+- For general information and documentation, click here: https://support.shotgunsoftware.com/entries/95441257
+- For information about Shotgun in general, click here: http://www.shotgunsoftware.com/toolkit
+
+## Using this engine in your Setup
+
+All the engines that are part of our standard engine suite are pushed to our engine Store.
```
Comment (rob-aitchison) [inline README.md]: "All the engines that are part of our standard configurations are pushed to our App Store."

File: README.md [hunk 3]
```diff
@@ -1,7 +1,22 @@
...
+This is where you typically go if you want to install an engine into a project you are
+working on. For an overview of all the engines and Engines in the Toolkit engine Store,
```
Comment (rob-aitchison) [inline README.md]: "For an overview of all the Apps and Engines in the Toolkit App Store"

File: README.md (hunk 2 above)
Comment (000paradox000) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (000paradox000) [reply README.md]: done

**Pull request merged**